npm install fails

Cannot run `npm install` as I get:

```
> aframe-inspector@0.9.1 prepublish C:\Users\vrig\Dev\repositories\aframe\aframe-inspector
> npm run dist
> aframe-inspector@0.9.1 dist C:\Users\vrig\Dev\repositories\aframe\aframe-inspector
> npm run dist:max && npm run dist:min
> aframe-inspector@0.9.1 dist:max C:\Users\vrig\Dev\repositories\aframe\aframe-inspector
> cross-env AFRAME_DIST=true npm run build
> aframe-inspector@0.9.1 build C:\Users\vrig\Dev\repositories\aframe\aframe-inspector
> NODE_ENV=production cross-env webpack --progress --colors
'NODE_ENV' is not recognized as an internal or external command,
operable program or batch file.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
```

--- a/package.json
+++ b/package.json
@@ -4,7 +4,7 @@
   "description": "A visual inspector tool for A-Frame.",
   "main": "dist/aframe-inspector.min.js",
   "scripts": {
-    "build": "NODE_ENV=production cross-env webpack --progress --colors",
+    "build": "cross-env NODE_ENV=production webpack --progress --colors",
     "deploy": "npm run ghpages",
     "dist": "npm run dist:max && npm run dist:min",
     "dist:max": "cross-env AFRAME_DIST=true npm run build",

